Week-5/Assignment5.4/calculater.test.js

```diff
@@ -0,0 +1,28 @@
+
+const mathOperations = require('./calculater');
+ 
+describe("Calculator tests", () => {
+ test('adding 1 + 2 should return 3', () => {
```

Comment:
[For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are string return "number are not integers".. add this test case as well..